packages/config-schema/src/schema/netlify-toml.schema.json

```diff
@@ -0,0 +1,722 @@
+      },
+      "properties": {
+        "deno_import_map": {
+          "title": "Import maps",
+          "description": "Relative path to an import map file to map module URLs to names.",
+          "type": "string"
+        },
+        "directory": {
+          "title": "Directory",
+          "description": "Custom path to your functions. The default location is `YOUR_BASE_DIRECTORY/netlify/functions`.",
+          "type": "string",
+          "default": "netlify/functions"
+        },
+        "external_node_modules": { "$ref": "#/definitions/FunctionConfigExternalNodeModules" },
+        "ignored_node_modules": { "$ref": "#/definitions/FunctionConfigIgnoredNodeModules" },
+        "included_files": { "$ref": "#/definitions/FunctionConfigIncludedFiles" },
+        "node_bundler": { "$ref": "#/definitions/FunctionConfigNodeBundler" }
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/configure-builds/file-based-configuration/#functions"
+        }
+      }
+    },
+    "edge_functions": {
+      "type": "array",
+      "default": [{ "path": "/path", "function": "function-name" }],
+      "items": {
+        "title": "Edge Functions",
+        "description": "While you can declare edge functions inline, you can use this section for more advanced edge function configurations.",
+        "type": "object",
+        "properties": {
+          "cache": {
+            "type": "string",
+            "description": "Should the response of the edge function be cached.",
+            "enum": ["off", "manual"],
+            "x-taplo": {
+              "docs": {
+                "enumValues": [
+                  "Default, edge function response is not cached.",
+                  "Edge function response is cached in the Netlify CDN based on the specified caching headers."
+                ]
+              }
+            }
+          },
+          "function": {
+            "type": "string",
+            "description": "Name of the function."
+          },
+          "path": {
+            "type": "string",
+            "description": "Path pattern to associate with this edge function.",
+            "pattern": "^/.*"
+          }
+        },
+        "additionalProperties": false,
+        "required": ["path", "function"]
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/edge-functions/declarations/"
+        }
+      }
+    },
+    "dev": {
+      "title": "Netlify Dev",
+      "description": "Netlify Dev enables a local development environment without any additional setup and this includes using the functions directory setting to scaffold and serve your functions locally. You can use the `[dev]` section to configure this local development environment. \n\nNote that `[dev]` doesn't run in the Bash shell, so you won't be able to use Bash-compatible syntax with the command.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
```

Comment:
I think some fields are still missing regarding the dev config from the CLI:
https://github.com/netlify/cli/blob/main/src/commands/dev/types.d.ts#L3-L22

like: `staticServerPort` or `envFiles` or `functions` and `live`